packages/hub-ui/src/features/badges/BadgeSection/BadgeSection.tsx

```diff
@@ -143,85 +97,140 @@ export const BadgeSection = (props: BadgeSectionProps) => {
             <div className="shrink-0 mt-auto ml-auto text-sm">
               <span className="font-semibold">
                 {collected.length}
-                {props.readonly ? "" : "/" + badges.length} Badges
+                {props.readonly ? "" : "/" + props.badges.length} Badges
               </span>
               <span className="ml-1">collected</span>
             </div>
           </div>
         </WidgetHeader>
         <WidgetBody>
           <div className="pt-2 pb-5 px-2">
-            {!props.readonly && (
-              <div className="p-2 flex w-full gap-x-2 text-zinc-600">
-                <div
-                  onClick={toggleShowMore}
-                  className="shrink-0 flex w-max ml-auto text-sm cursor-pointer text-zinc-400 hover:text-zinc-500"
-                >
-                  <span>Show more</span>
-                  <div className="inline-block ml-1 overflow-hidden h-5 w-5">
-                    <Icon name="up & down arrow" />
-                  </div>
```

Comment:
@perceeg , one question we do not need show more button ?

Reply:
Correct - end-user no longer needs the ability to show less/more badges on their home page. This is a legacy requirement.